homeassistant/components/teslemetry/const.py

```diff
@@ -54,3 +54,20 @@ class TeslemetryClimateSide(StrEnum):
 
     DRIVER = "driver_temp"
     PASSENGER = "passenger_temp"
+
+
+class TeslemetryEnum:
+    """Helper class to handle options for protobuf enums."""
```

Comment:
Whats so special about protobuf enums?

Reply:
But wouldn't that make it library specific in a way?

Reply:
Converting to a lowercase string for translations keys and an options list without "Unknown" isnt really generic, Home Assistant is the only place that would get used.

Reply:
Check

Reply:
Ad discussed on Discord, this is now in the library in its correct case with Unknown, so that can be dealt with in Home Assistant as required.